nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/blob/ColdStorageHelper.java

```diff
@@ -75,6 +83,61 @@ public static DocumentModel moveContentToColdStorage(CoreSession session, Docume
         return session.saveDocument(documentModel);
     }
 
+    /**
+     * Requests a retrieval of the cold storage content associated with the document of the given {@link DocumentRef}.
+     *
+     * @param session the core session
+     * @param documentRef the document reference
+     * @param numberOfDaysOfAvailability number of days that you want your cold storage content to be accessible after
+     *            restoring
+     * @apiNote This method will initiate a restoration request, calling the {@link Blob#getStream()} during this
+     *          process doesn't mean you will get the blob's content.
+     * @return the updated document model if the retrieve succeeds
+     * @throws NuxeoException if there is no cold storage content associated with the given document, or if it is being
+     *             retrieved
+     */
+    public static DocumentModel requestRetrievalFromColdStorage(CoreSession session, DocumentRef documentRef,
+            int numberOfDaysOfAvailability) {
```

Comment:
Let's make `numberOfDaysOfAvailability` a `Duration` directly in the signature instead of an `int`, and use durations everywhere we can.